<launch>
  <node pkg="icvfollower" name="icv_carla_control"  type="icv_carla_control" output="screen">
  <param name="~target_speedxp" type="double" value="4"/>   
  <param name="~carla_flag" type="int" value="0"/>
  </node> 
  <node pkg="xpmotors" name="auto_state"  type="auto_state_node" output="screen">
  </node> 
 <node pkg="xpmotors" name="auto_control"  type="auto_control_node" output="screen">
  <rosparam file="$(find xpmotors)/conf.yaml"/>
  </node> 

  
</launch>
